src/client/components/NavBar.tsx

```diff
@@ -1,9 +1,11 @@
 import React from "react";
 import { Link } from "react-router-dom";
 
-interface NavBarProps {}
+interface NavBarProps {
+}
```

Comment:
Potentially yes, that's a constant over my code base that props are initialized like that to control the typing. I'll go back through on a refactor at the end to clean these up if they're not needed. I'd rather have them here to be available because I'm not sure if I'll remember to add them when I add props if I do 😅

Reply:
I've just added a ticket to my project to remind me that this is something I should look for in a late refactor